### PR TITLE
Allow using a file:// url for any SCAP guides pushed on the media

### PR DESCRIPTION
**Pull request by jcpunk**
This allows for a bit more flexible resource locations and the ability to put alternate guides on the media.

### CODE REVIEW

File: org_fedora_oscap/data_fetch.py [hunk 1]
```diff
@@ -90,8 +96,7 @@ def fetch_data(url, out_file, ca_certs=None):
     out_dir = os.path.dirname(out_file)
     utils.ensure_dir_exists(out_dir)
 
-    if url.startswith("http://") or url.startswith("https://") \
-       or url.startswith("ftp://"):
+    if can_fetch_from(url):
         _fetch_http_ftp_data(url, out_file, ca_certs)
```
Comment (vpodzime) [inline org_fedora_oscap/data_fetch.py]: Probably a time for a better name for this function. Could you please update the pull request with such a change? Maybe something like ``_curl_fetch()``?

**Comment by jcpunk** [comment]
I can rename that function, but it comes out of code already within the project (data_fetch line 55).  I just want to be sure that is noted before taking a public function and renaming it.

**Comment by vpodzime** [comment]
> I can rename that function, but it comes out of code already within the project (data_fetch line 55). I just want to be sure that is noted before taking a public function and renaming it.

I'm not sure we are speaking about the same thing here. I meant to rename the private ``_fetch_http_ftp_data()`` function to ``_curl_fetch()``.

**Comment by jcpunk** [comment]
Ahh, got it.

New data pushed

**Comment by vpodzime** [comment]
Looks good to me now, thanks!

**Pull request merged**